lib/ansible/modules/windows/win_scheduled_task.ps1

```diff
@@ -25,6 +25,8 @@ $ErrorActionPreference = "Stop"
 $params = Parse-Args $args;
 
 $days_of_week = Get-AnsibleParam $params -name "days_of_week"
+$interval = Get-Attr $params -name "interval" -default $null
+$interval_unit = Get-Attr $params -name "interval_unit" -default minutes
 $enabled = Get-AnsibleParam $params -name "enabled" -default $true
 $enabled = $enabled | ConvertTo-Bool
 $description = Get-AnsibleParam $params -name "description" -default " "
```

Comment:
Will fix `Get-Attr` -> `Get-AnsibleParam`.

Description probably defaults to "" because it is unnecessary to make it required and original dev's didn't want to add support for checking if description field exists.  Personally, I think it's better to leave it as is since it honestly doesn't matter what the description is.